app/code/Magento/ImportExport/Test/Unit/Model/Import/Source/FileParser/_files/test_options.csv

```diff
@@ -0,0 +1 @@
+column1|;column2;|column3
```

Comment:
why do we need to use CSV files for Unit testing? 

Integration tests are made for these purposes